icu4j/main/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+* © 2025 and later: Unicode, Inc. and others.
+* License & terms of use: http://www.unicode.org/copyright.html
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.ibm.icu</groupId>
+    <artifactId>icu4j-root</artifactId>
+    <version>79.0.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>icu4j-main</artifactId>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>framework</module>
+    <module>core</module>
+    <!-- For publishing to Maven Central -->
+    <module>charset</module>
+    <module>collate</module>
+    <module>currdata</module>
+    <module>langdata</module>
+    <module>regiondata</module>
+    <module>translit</module>
+    <module>common_tests</module>
```

Comment:
Since we're refactoring things anyways, can we make module names consistent? Can we use dashes (-) instead of underscores (_) ? I think the relevant modules are non-public anyways, so it should be safe to do.

Reply:
> Can we use dashes (-) instead of underscores (_) 

That would be highly disruptive.
The module names here are in fact folder names.
Changing them means moving tons of Java files around.

If anything I would go the other way, to use `_` everywhere.
Dashes are problematic because you can't use them in namespaces, folders inside google3.

Note that the `<module>` in maven lingo (in `pom.xml`) and the Java `module` (introduced in JDK 9) are completely separate concepts. The maven one precedes the Java ones by many years.

Maven 4 changes the name to subprojects (`module` works but is deprecated):
https://maven.apache.org/whatsnewinmaven4.html#Modules_are_now_subprojects

Reply:
Yes, I was using "module" in the Maven sense. Okay, that explains it -- we have a constraint on a large downstream user whose codebase (or rather, build tool thereof) is based on Python, which turns filenames into identifiers, and the constraint about dashes in identifiers leaks back into the filename constraints, and all the way up to ICU. That's fine, at least I know the reason.